content/_index.md

```diff
@@ -7,12 +7,11 @@ extra:
   menu_title: Home
 ---
 
-**Organic Maps** is a free Android & iOS offline maps app for travelers, tourists, hikers, drivers and cyclists based on **[OpenStreetMap][openstreetmap]** data created by the community.
-It is a privacy-focused, open-source [fork][fork] of **Maps.me** app (previously known as [**MapsWithMe**][mapswithme]), maintained by the same people who created **MapsWithMe** in 2011.
+**Organic Maps** is a privacy-first offline maps & GPS app for hiking, cycling, biking, and driving. Absolutely free. No ads. No tracking. Developed with love by the open-source community. Powered by [OpenStreetMap][openstreetmap] data.
```

Comment:
Just not sure if OM could be called "privacy-first". "Privacy-focused" - yes.
See https://github.com/orgs/organicmaps/discussions/8559#discussioncomment-9868694 for details.

Reply:
Updated